apps/web/vibes/soul/components/product-list-carousel/style.css

```diff
@@ -0,0 +1,7 @@
+.no-scrollbar::-webkit-scrollbar {
+  display: none;
+}
+.no-scrollbar {
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
```

Comment:
Should we include this globally?

Reply:
I think including this globally would be helpful!